statistics/histogram.go

```diff
@@ -581,7 +582,7 @@ func (idx *Index) equalRowCount(sc *stmtctx.StatementContext, b []byte) (float64
 func (idx *Index) getRowCount(sc *stmtctx.StatementContext, indexRanges []*ranger.IndexRange) (float64, error) {
 	totalCount := float64(0)
 	for _, indexRange := range indexRanges {
-		indexRange.Align(len(idx.Info.Columns))
+		isPoint := len(indexRange.LowVal) == len(indexRange.HighVal) && len(indexRange.LowVal) == len(idx.Info.Columns)
```

Comment:
Is `isPoint` a little miss leading? Maybe `isFullRange` is better. Also, you can put this declaration near where it is used.